lib/app/person.ex

```diff
@@ -1,45 +1,74 @@
 defmodule App.Person do
   use Ecto.Schema
-  import Ecto.Changeset
+  import Ecto.{Changeset, Query}
   alias App.Repo
-  # https://stackoverflow.com/a/47501059/1148249
   alias __MODULE__
 
   schema "people" do
-    field :status, :id
-    field :tag, :id
+    field :auth_provider, :string
+    field :givenName, Fields.Encrypted
+    field :key_id, :integer
+    field :locale, :string
+    field :picture, Fields.Encrypted
+    field :status, :integer
+
     timestamps()
   end
 
-  @doc """
-  Default attributes validation for Person
-  """
+  def get_person!(id), do: Repo.get!(__MODULE__, id)
+
+  @doc false
   def changeset(person, attrs) do
     person
-    |> cast(attrs, [:id, :status, :tag])
+    |> cast(set_id(attrs), [
+      :givenName,
+      :auth_provider,
+      :picture,
+      :locale,
+      :status,
+      :id
+    ])
+    |> validate_required([:givenName, :auth_provider])
   end
 
-  def create_person(attrs) do
-    changeset(%Person{}, attrs)
-    |> Repo.insert!()
+  # workaround for people_pkey (unique_constraint)
```

Comment:
This is the workaround for avoiding primary key conflicts described in https://github.com/dwyl/app-mvp/issues/115#issuecomment-1206190867

Reply:
Removing the `people` schema/table completely will eliminate this "hacky" code. #118 🎉